cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -200,7 +200,9 @@
             {% endfor %}
         {% endif %}
         {% if 'filterable-list' in controls.form_type and posts is defined %}
-            {% if posts.paginator.count == 0 %}
+            {% if not request.GET %}
```

Comment:
@sebworks request.GET is empty if there are no filters specified, which is the condition we want to be handling here. To `notification`, we pass in `false` as one of the params, so that it does not get displayed.